### PR TITLE
NoMachine.NoMachine version 8.5.3

### PR DESCRIPTION
**Pull request by fujii**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/108891)

### CODE REVIEW

File: manifests/n/NoMachine/NoMachine/8.5.3_1/NoMachine.NoMachine.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: NoMachine.NoMachine
+PackageVersion: 8.5.3_1
```
Comment (mdanish-kh) [inline manifests/n/NoMachine/NoMachine/8.5.3_1/NoMachine.NoMachine.installer.yaml:5]:
```suggestion
PackageVersion: 8.5.3
```
Must match App&Features / Control Panel

File: manifests/n/NoMachine/NoMachine/8.5.3_1/NoMachine.NoMachine.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: NoMachine.NoMachine
+PackageVersion: 8.5.3_1
```
Comment (mdanish-kh) [inline manifests/n/NoMachine/NoMachine/8.5.3_1/NoMachine.NoMachine.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 8.5.3
```
Must match App&Features / Control Panel

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
After committing the suggested changes, you would also have to rename the version folder to match the new PackageVersion. The folder under `manifests/n/NoMachine/NoMachine` should be renamed from `8.5.3_1` -> `8.5.3`

A quick and easy way to do this would be here in GitHub Web by pressing the <kbd>.</kbd> (dot) key on your keyboard that'll open up the PR in VS Code Online. Navigate to Explorer menu, find the required folder and make the appropriate rename changes.

After that, stage and commit your changes from Source Control menu on the left. (The changes are automatically pushed)

File: manifests/n/NoMachine/NoMachine/8.5.3_1/NoMachine.NoMachine.installer.yaml (hunk 1 above)
Comment (fujii) [reply manifests/n/NoMachine/NoMachine/8.5.3_1/NoMachine.NoMachine.installer.yaml]: Fixed.

File: manifests/n/NoMachine/NoMachine/8.5.3_1/NoMachine.NoMachine.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: NoMachine.NoMachine
+PackageVersion: 8.5.3_1
```
[the remark this answers is not in the document]
Comment (fujii) [reply manifests/n/NoMachine/NoMachine/8.5.3_1/NoMachine.NoMachine.yaml]: Fixed.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**